### ISSUE TITLE
Clock sometimes appears, sometimes not

### ISSUE THREAD

**Issue by buscon** [supercollider/supercollider#4202]
I notice an irregular appearance of the clock on the bottom left corner of the window, sometimes is there, sometimes not.

Environment
-----------

* Your SuperCollider version:
SC 3.10
* Your operating system and version:

Linux Mint 19, tara (same as Ubuntu Bionic)

Steps to reproduce (for bugs)
-----------------------------
In an apparently random way, the clock appears or not on the left bottom corner.

**Comment by buscon** [comment]
I don't think so, it does really look integrated into the SC gui, here is a screenshot:
![sc_clock](https://user-images.githubusercontent.com/2972257/49701910-86619680-fbf2-11e8-9810-6ac2642dc81e.png)

**Comment by patrickdupuis** [comment]
Wow, I've never seen that. I think it doesn't show at all on some DE's. What are you using?

**Comment by buscon** [comment]
I am on Linux Mint with Mate. It is actually a lightweight DE. I was surprised about that too, but as I mentioned above, it seems to be there only sometimes...

**Comment by kenokenobingo** [comment]
I suppose it’s the same on macOS.

**Comment by jamshark70** [comment]
Just glancing through the IDE issue list... Grepping for "clock" finds a ClockStatusBox class defined in ./widgets/main_window.cpp (line 1591 ff.). But I can't find any menu command or editor preference to enable or disable its display.

It looks like it's supposed to be enabled by full-screen display:

```
void MainWindow::updateClockWidget(bool isFullScreen) {
    if (!isFullScreen) {
        if (mClockLabel) {
            delete mClockLabel;
            mClockLabel = NULL;
        }
    } else {
        if (mClockLabel == NULL) {
            mClockLabel = new ClockStatusBox(this);
            statusBar()->insertWidget(0, mClockLabel);
        }
    }
}
```

But for whatever reason, on my desktop (XFCE), setting the window to full screen does not activate the clock widget.

I wonder what would be the harm in simplifying the logic and just displaying the clock all the time...?

I think I understand the rationale -- if the window is full screen, then the system's clock display is hidden, so it would be nice to have something in *our* window. But it doesn't work consistently.
